package.json

```diff
@@ -57,47 +57,47 @@
   "license": "MIT",
   "dependencies": {
     "@improbable-eng/grpc-web": "^0.12.0",
-    "@textile/threads-client-grpc": "^0.1.12",
-    "@textile/threads-core": "0.0.3-alpha.2",
+    "@textile/threads-client-grpc": "^0.1.13",
+    "@textile/threads-core": "0.0.3-alpha.3",
     "@types/google-protobuf": "^3.7.2",
     "b64-lite": "^1.4.0",
     "bs58": "^4.0.1",
     "google-protobuf": "^3.10.0",
     "isomorphic-ws": "^4.0.1",
-    "uuid": "^3.3.3",
-    "ws": "^7.2.0"
+    "uuid": "^7.0.2",
+    "ws": "^7.2.3"
   },
   "devDependencies": {
     "@types/b64-lite": "^1.3.0",
     "@types/bs58": "^4.0.1",
-    "@types/chai": "^4.2.5",
+    "@types/chai": "^4.2.11",
     "@types/mocha": "^7.0.1",
     "@types/multibase": "^0.6.0",
-    "@types/uuid": "^3.4.6",
-    "@typescript-eslint/eslint-plugin": "^2.9.0",
-    "@typescript-eslint/parser": "^2.9.0",
+    "@types/uuid": "^7.0.2",
+    "@typescript-eslint/eslint-plugin": "^2.26.0",
+    "@typescript-eslint/parser": "^2.26.0",
     "chai": "^4.2.0",
     "eslint": "^6.7.1",
     "eslint-config-prettier": "^6.7.0",
     "eslint-config-strict": "^14.0.1",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.1",
     "jsdoc": "^3.6.3",
-    "libp2p-crypto": "^0.17.3",
-    "mocha": "^6.2.2",
+    "libp2p-crypto": "^0.17.5",
```

Comment:
So we still need this as a top-level dependency, or is this left-over from when it was missing in threads-core?